helm/thingsboard/values.yaml

```diff
@@ -327,6 +327,8 @@ postgresInitDB:
     check:
       # default value of the timeout for the pg_isready command
       timeout: 3
+      # number of partitions we want to set in the queue table
+      partitions: 2
```

Comment:
Keep it 10 by default I think. We should also set the TP_CORE partitions to the same value for both MQTT and tb-node services. This is still done via env var I think.